app/controllers/forgot_password_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ForgotPasswordController < ApplicationController
+  def show
+    if session[:email].blank?
+      redirect_to new_user_password_path
+    else
+      @resend_confirmation = params[:resend].present?
+      email = session.delete(:email)
+
+      @password_reset_email_form = PasswordResetEmailForm.new(email)
+
+      render :show, locals: { email: email }
```

Comment:
style-wise, is it more common to assign to `@email` and remove the explicit `render` call since it will be called implicitly?